modules/ncov-tools.nf

```diff
@@ -36,6 +36,26 @@ process download_artic_ncov2019 {
   """
 }
 
+process download_ncov_watchlists {
+  tag { version }
+  executor 'local'
+  
+  input:
+  val(version)
+  
+  output:
+  tuple path("watchlists/watchlists.csv"), path("watchlists", type: 'dir')
+
+
+  script:
+  """
+  wget https://github.com/dfornika/ncov_watchlists/archive/v${version}.tar.gz
```

Comment:
I'm just using this repo as a placeholder. I'll replace it with a repo under the [BCCDC-PHL](https://github.com/BCCDC-PHL) org once we settle on a place to store our watchlists.